website/docs/reference/actions.md

```diff
@@ -2,11 +2,13 @@
 title: Actions
 ---
 
-:::info Availability
+:::note Availability
 
-Actions were introduced as a beta feature in **Unleash 5.11** and are only available in Unleash Enterprise.
+
+**Plan**: [Enterprise](https://www.getunleash.io/pricing) | **Version**: `5.11+` in BETA
```

Comment:
Didn't this go GA? Or is it still in beta?

Reply:
Yep, still in beta